src/AllTravelers.js

```diff
@@ -10,46 +10,51 @@ class AllTravelers {
   getTrips(theStatus) {
     if (theStatus === 'pending') {
       return this.trips.filter(trip => trip.date >= moment().format('YYYY/MM/DD') && trip.status === 'pending').map(trip => {
-        const tripsObj = {};
-        tripsObj.name = this.travelers.find(traveler => traveler.id === trip.userID).name;
-        tripsObj.date = trip.date;
-        tripsObj.duration = trip.duration;
-        tripsObj.travelers = trip.travelers;
-        tripsObj.destination = this.destinations.find(dest => dest.id === trip.destinationID).destination;
-        tripsObj.status = trip.status;
-        return tripsObj;
+        const tripsDetails = {};
+        tripsDetails.name = this.travelers.find(traveler => traveler.id === trip.userID).name;
+        tripsDetails.date = trip.date;
+        tripsDetails.duration = trip.duration;
+        tripsDetails.travelers = trip.travelers;
+        tripsDetails.destination = _.find(this.destinations, dest => dest.id === trip.destinationID).destination;
+        tripsDetails.image = this.destinations.find(dest => dest.id === trip.destinationID).image;
+        tripsDetails.alt = this.destinations.find(dest => dest.id === trip.destinationID).alt;
+        tripsDetails.status = trip.status;
+        return tripsDetails;
       })
     } else {
       return this.trips.map(trip => {
-        const tripsObj = {};
-        tripsObj.name = this.travelers.find(traveler => traveler.id === trip.userID).name;
-        tripsObj.date = trip.date;
-        tripsObj.duration = trip.duration;
-        tripsObj.travelers = trip.travelers;
-        tripsObj.destination = this.destinations.find(dest => dest.id === trip.destinationID).destination;        
-        tripsObj.status = trip.status;
-        return tripsObj;
+        const tripsDetails = {};
```

Comment:
Since you have the same logic here as lines 13-22, could you pull this into a separate function and call that function inside of your if/else statement?